api/client.yaml

```diff
@@ -1609,7 +1886,7 @@ components:
           type: array
           minimum: 1
           items:
-            $ref: '#/components/schemas/CreateCustomerAddress'
```

Comment:
This `$ref` needs changed across all uses of it.

Reply:
Agreed, and they are